### ISSUE TITLE
New Command/ Flag to notify or download latest updates to an already pulled model. 

### ISSUE THREAD

**Issue by nviraj** [ollama/ollama#5751]
There might be scenarios where quants might be recreated (e.g. gemma) or templates could be updated in the model registry. 

If there is some way to show it during ls or pull commands as info, it can improve user experience. Users can benefit from better quantization and prompt templates accordingly.

**Comment by chigkim** [comment]
I'd also love to see something like this:
`ollama update --list`
Show all the models that can be updated.
`ollama update --all`
Pull all the updates!

**Comment by nkeilar** [comment]
Yes, this is long overdue. It happens frequently that you download a 40GB model, one day later a version is revised and the only way to find out is by chance you check the model hash manually. I have little confidence that any of my models are the latest version, but I really have no idea unless one by one I check.
